.travis.yml

```diff
@@ -69,7 +69,7 @@ before_script:
     # Make sketchbook/libraries folder.
     - mkdir -p $CI_SRC_PATH/sketchbook/libraries
     - export CI_LIB_PATH=$CI_SRC_PATH/sketchbook/libraries
-    # Make rosserial_arduino in the sketchbook/libraries folder, including victoria_nav_msgs
+    # Make rosserial_arduino in the sketchbook/libraries folder, including victoria_nav_msgs,victoria_sensor_msgs
     - rosrun rosserial_arduino make_libraries.py $CI_LIB_PATH victoria_nav_msgs
```

Comment:
I see you comment in email, but not here online.  Annoying.  But I think the answer is that

source ~/catkin_ws/devel/setup.bash

includes the victoria_msgs and the make_libraries.py just sucks in everything in the workspace. If I leave off both of the packages, I still see them generated in the ros_lib.  So, I guess I didn't have to include anything in the first place.  I guess I'll remove them.